Documentation/ceph-pool-crd.md

```diff
@@ -88,6 +134,15 @@ When creating an erasure-coded pool, it is highly recommended to create the pool
   * `target_size_ratio:` gives a hint (%) to Ceph in terms of expected consumption of the total cluster capacity of a given pool, for more info see the [ceph documentation](https://docs.ceph.com/docs/master/rados/operations/placement-groups/#specifying-expected-pool-size)
   * `compression_mode`: Sets up the pool for inline compression when using a Bluestore OSD. If left unspecified does not setup any compression mode for the pool. Values supported are the same as Bluestore inline compression [modes](https://docs.ceph.com/docs/master/rados/configuration/bluestore-config-ref/#inline-compression), such as `none`, `passive`, `aggressive`, and `force`.
 
+* `mirrored`: Sets up mirroring of the pool
```

Comment:
`mirrored` just fits better with what we have today: `replicated`, `erasureCoded` so it's probably better to stay consistent.

Reply:
What makes the most sense for this feature? That's more important than to follow the same pattern as the other settings. I'd vote that these are the "mirroring" settings, rather than the "mirrored" settings.

Reply:
Ok if you feel it's more natural, I'll change it.